models/offer.go

```diff
@@ -38,6 +42,37 @@ const enabledOffers = `
 		AND offers.enabled = true
 `
 
+func buildScope(enabledOffers string, filterAttrs map[string]string) string {
+	subQueries := []string{enabledOffers}
+	for k, v := range filterAttrs {
+		// TODO: Possible SQL injection
```

Comment:
My bad, you are totally correct. Well in this case it is a serious vulnerability for "eq" and "neq" operators because for the other ones we are checking that they are floats, right? Maybe we could be stricter while validating "eq" and "neq" operators values (don't allow pg keywords, for example)